### PR TITLE
oidc: Expose Issuer on the OIDC Provider struct

### PR DESCRIPTION
**Pull request by pquerna**
- I have a couple places where i'm keeping a Provider around in other structs, and I'd like to get at the original Issuer, seems easy to just expose this as another method?

### CODE REVIEW

**Comment by ericchiang** [comment]
@pquerna we exported it previously. Want to just do that?

**Comment by pquerna** [comment]
I'm easy, it seemed intentional that *Provider didn't export any fields, so was jus trying to keep the pattern

**Comment by ericchiang** [comment]
> it seemed intentional that *Provider didn't export any fields,

Yep, I'd just prefer fields to accessers :) I think we're fine to unexport any required metadata claims (just like the IDToken).

File: oidc.go [hunk 1]
```diff
@@ -143,6 +143,11 @@ func (p *Provider) Endpoint() oauth2.Endpoint {
 	return oauth2.Endpoint{AuthURL: p.authURL, TokenURL: p.tokenURL}
 }
 
+// Issuer returns the OAuth2 issuer claim for the given provider.
+func (p *Provider) Issuer() string {
+	return p.issuer
```
Comment (ericchiang) [inline oidc.go]: As a follow up, can we just make this field exported?

**Comment by ericchiang** [comment]
Going to close this for now. Workaround seems like:

```go
type provider struct {
    *oidc.Provider

    IssuerURL string
}
```

**Pull request closed without merging**